external/wycheproof/test.properties

```diff
@@ -4,7 +4,4 @@ home_path="\${WORKDIR}"
 test_results="testResults"
 tag_version="master"
 bazel_version="1.2.1"
-debian_packages="git wget unzip zip g++"
-alpine_packages="bash git wget unzip zip g++ bash"
-centos_packages="git wget unzip zip gcc-c++"
-ubi_packages="git wget unzip zip gcc-c++"
+ubuntu_packages="git wget unzip zip gcc-c++"
```

Comment:
Did you run grinder changing ubuntu_packages from `git wget unzip zip g++` to  `git wget unzip zip gcc-c++`?

Reply:
thanks for the suggestion , going to double check it too

Reply:
![console](https://user-images.githubusercontent.com/87583038/142575728-4a1abd66-8bdd-4796-85d7-b8d5c58b0014.png)
as per this grinder https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/2570/console great catch!
Clearing it up for all in a new branch and PR